\documentclass[fleqn,11pt]{wlpeerj}
\usepackage{lineno}
\usepackage{url}

\title{The ecological genomics of gypsy moth invasion along a latitudinal
gradient}

\author[1]{Christopher J. Friedline}
\author[1]{Trevor M. Faske}
\author[1]{Erin M. Hobson}
\author[2]{Dylan Parry}
\author[1]{Derek M. Johnson}
\author[3]{Lily M. Thompson}
\author[3,*]{Kristine L. Grayson}
\author[1,*,\textdagger]{Andrew J. Eckert}
\affil[1]{Department of Biology, Virginia Commonwealth University}
\affil[2]{College of Environmental Science and Forestry, State University of
New York}
\affil[3]{Department of Biology, University of Richmond}
\affil[*]{Author contributed equally}
\affil[ \textdagger]{Corresponding author}

\keywords{}

\begin{abstract}
TODO
\end{abstract}

\begin{document}

\flushbottom
\maketitle
\thispagestyle{empty}

\linenumbers%

\section*{Introduction}
TODO


\section*{Methods}

\subsection*{Library preparation and sequencing}

Genomic DNA extracted using Qiagen DNeasy Plant Mini kit (Qiagen, cat \# 69104)
according to manufacturer's protcool.  The library was prepared and sequenced as
a single run of paired-end reads on the Illumina HiSeq 2500 platform at the
Virginia Commonwealth Univesrity Nucleic Acids Research Facility. Quality was
initially asssed using FastQC \citep{fastqc} and subsequently processed with
IPython~\citep{Perez:2007hy} and BioPytho~\citep{Cock:2009hj}. The read pairs
were each evaluated along sliding windows of 5 bp.  If the mean score in this
window was below a score of 30, the read was trimmed at that beginning of the
window. If the shortened read was less than 50\% of the length of the original
read, it was discarded along with its pair. Additionally, if 20\% of the bases
in a read had quality values less than 30, it was discarded along with its pair.

\subsection*{Draft assembly}

An assembly was produced for the sole purpose of variant calling and is not
considered as a candidate for complete genomic assembly. The quality-filtered
reads from the paired-end sequencing library was used as input into
\texttt{MaSuRCA} \citep[][version 2.3.2]{Zimin:2013kn}, with the following
parameters changed from the defaults: mean/standard deviation of read length:
400/60, use linking mates: 1: cgwErrorRate: 0.15.  Quality of the assembly was
assessed using the stats generated by the assembly process as well as with the
\texttt{BUSCO} \citep[][version 1.1b1]{Simao:2015kk} toolset. For
\texttt{BUSCO}, the software was built automatically using a Docker image (see
GitHub repository information below). The assembly was evaluated against
pre-computed \texttt{Augustus} \citep{Stanke:2003eo} metaparameters of  three
species (\textit{Aedes}, \textit{Heliconius}, and \textit{Drosophila}) and the
BUSCO Arthropoda lineage profile using a \texttt{tblastn} e-value cutoff of
$0.001$.

\subsection*{Sequence processing and variant calling} Sequencing of 192
individual moths, exclusive of the one used for assembly,  as performed on
reduced genomic DNA per~\cite{PARCHMAN:2012ca}. In total, two  Illumina HiSeq
2500 lanes were sequenced. Bases with quality scores less than 33 were masked,
per~\cite{Yun:2014dn},  using \texttt{fastq\_masker} from the FastX toolkit
\citep[][version 0.0.14]{citeulike:9103573}.

Masked fastq files were mapped using \texttt{bowtie2}~\citep[][version
2.2.4]{Langmead:2012jh}, using flags  \texttt{--local
--very-sensitive-local}.  The resulting  sam files were converteted to their
binary equivalent, sorted, and indexed. To each of these indexed bam files,
appropriate \texttt{RG} ID and sample information was added using
\texttt{picard} \\ (\url{https://broadinstitute.github.io/picard}) version 1.112
to inform variant calling.

Duplicates were marked using \texttt{Picard} and the resulting files were
indexed using \texttt{samtools} \citep[][version 1.2]{Li:2009ka}.  Sequence
variants were called using samtools and bcftools \citep[][version
1.2]{Li:2009ka} with individual  ploidy level set to \texttt{2} for all samples.
The variants were filtered  using \texttt{vcftools}~\citep[][version
0.1.14]{Danecek:2011gz}, such that only biallelic-SNPs remained in cases where a
variant call was present in at least 50\% of the samples and the minimum
gentoype quality (\texttt{--minQG}) of 3. An additional filtering using
\texttt{vcfutils.pl} (from \texttt{bcftools}) was performed to only keep sites
with mapping qualities of at least 10 (\texttt{varFilter -Q 10}). The vcf file
was then double checked to ensure all reference and alternate alleles were
single nucleotides before imputation with \texttt{Beagle}
\citep[][version 4.0]{Browning:2007ge}.  The impuation was performed on 30 CPUs
using the following non-default parameters (\texttt{phase-its=10, burnin-its=20,
impute-its=10}).

For each set of imputed (IMP) and non-imputed (NI) SNPs, the following filtering
was performed.  First, \texttt{VCFtools} was used to restrict SNPs to the
following criteria: only biallelic SNPs
(\texttt{--min-alleles/--max-alleles=2}), the minor allele must be found in at
least one sample (\texttt{--mac=1}), the minor allele frequency must be at least
1\% (\texttt{--mac=0.01}).  Depth across samles (DP), Alternate allele call quality (QUAL),
and RMS mapping quality values (MQ), reported in the VCF file were examined.  An additional
filtering step was conducted using \texttt{vcfutils.pl} (from \texttt{bcftools}) to
filter SNPs with MQ values less than 10.




monomorphic SNPs were removed, all SNPs with minor allele frequencies (corrected
for finite population size) $< 0.01$ were removed, SNPs with $F_{IS} < -0.05$ or
$F_{IS} > 0.5$ were removed.  We only considered SNPs to be valid if they passed
filtering in both the IMP and NI datasets. Finally, SNPs were oriented according
to dosage of globally-minor allele, rather than alternate allele with respect to
the assembly. In other words, in a file of genotypes where 0/1/2 would normally
represent counts of alternate allele in a diploid genotype,  (as output by
\texttt{vcftools}), 0/1/2 were recoded to represent counts of the minor allele
(i.e., a genotype of 2 indicates two copies of the globally minor alele). This
``swapped'' representation was then used as input for downstream analysis.


\subsection*{Population statistics}
To account for population structure in the data,


\subsection*{Environmental assocation}

\subsection*{Loci under selection}


\subsection*{}

\section*{Results}
TODO


\section*{Discussion}
TODO


\section*{Conclusion}
TODO


\section*{Figures}
TODO

\section*{Tables}
TODO

\section*{Supplementary material}
TODO

\clearpage
\bibliography{refs}

\end{document}
