\documentclass[fleqn,10pt]{wlpeerj}
\usepackage{natbib}

\title{The ecological genomics of gypsy moth invasion along a latitudinal gradient}

\author[1]{Christopher J. Friedline}
\author[1]{Trevor M. Faske}
\author[1]{Erin M. Hobson}
\author[2]{Dylan Parry}
\author[1]{Derek M. Johnson}
\author[3]{Lily M. Thompson}
\author[3,*]{Kristine L. Grayson}
\author[1,*,\textdagger]{Andrew J. Eckert}
\affil[1]{Department of Biology, Virginia Commonwealth University}
\affil[2]{College of Environmental Science and Forestry, State University of New York}
\affil[3]{Department of Biology, University of Richmond}
\affil[*]{Author contributed equally}
\affil[ \textdagger]{Corresponding author}

\keywords{}

\begin{abstract}
TODO
\end{abstract}

\begin{document}

\flushbottom
\maketitle
\thispagestyle{empty}

\section*{Introduction}
TODO


\section*{Methods}

\subsection*{Library preparation and sequencing}

Genomic DNA extracted using Qiagen DNeasy Plant Mini kit (Qiagen, cat \# 69104)
according to manufacturer's protcool.  The library was prepared and sequenced as
a single run of paired-end reads on the Illumina HiSeq 2500 platform at the
Virginia Commonwealth Univesrity Nucleic Acids Research Facility. Quality was
initially asssed using FastQC \citep{fastqc} and subsequently processed with
IPython \cite{Perez:2007hy} and BioPython \cite{Cock:2009hj}. The read pairs
were each evaluated along sliding windows of 5 bp.  If the mean score in this
window was below a score of 30, the read was trimmed at that beginning of the
window. If the shortened read was less than 50\% of the length of the original
read, it was discarded along with its pair. Additionally, if 20\% of the bases
in a read had quality values less than 30, it was discarded along with its pair.

\subsection*{Draft assembly}

An assembly was produced for the sole purpose of variant calling and is not
considered as a candidate for complete genomic assembly. The quality-filtered
reads from the paired-end sequencing library was used as input into MaSuRCA
\citep{Zimin:2013kn}, version 2.3.2, with the following parameters changed from
the defaults: mean/standard deviation of read length: 400/60, use linking mates:
1: cgwErrorRate: 0.15.  Quality of the assembly was assessed using  the stats
generated by the assembly process as well as with the BUSCO \citep{Simao:2015kk}
toolset. For BUSCO, the software was built automatically using a Docker image
(see GitHub repository information below). The assembly was evaluated against
pre-computed Augustus \citep{Stanke:2003eo} metaparameters of  three species
(\textit{Aedes}, \textit{Heliconius}, and \textit{Drosophila}) and the BUSCO
Arthropoda lineage profile using a tblastn e-value cutoff of $0.001$.

\subsection*{Sequence processing and variant calling}


\subsection*{Population statistics}

\subsection*{Environmental assocation}

\subsection*{Loci under selection}


\subsection*{}

\section*{Results}
TODO


\section*{Discussion}
TODO


\section*{Conclusion}
TODO


\section*{Figures}
TODO

\section*{Tables}
TODO

\section*{Supplementary material}
TODO

\clearpage
\bibliography{refs}

\end{document}
